Docker/Production/entando-full-stack/appbuilder/build.sh

```diff
@@ -1,15 +1,5 @@
 #!/usr/bin/env bash
 export VERSION=${1:-5.0.1-SNAPSHOT}
-echo $VERSION
-export DOCKER_TAG=$VERSION
-export IMAGE_NAME="entando/appbuilder:$VERSION"
-export DOCKERFILE_PATH="$(dirname $0)/Dockerfile"
-source hooks/build
-if [ $? -eq 0 ]; then
-    docker login -u $(oc whoami) -p $(oc whoami -t) 127.0.0.1:5000
-    docker tag $IMAGE_NAME 127.0.0.1:5000/$IMAGE_NAME
-    docker push 127.0.0.1:5000/$IMAGE_NAME
-else
-    echo "Docker build failed"
-fi
+export ENTANDO_IMAGE="appbuilder"
 
+source ../../../../common/build-common.sh
```

Comment:
Just to verify this relative path is correct and supposed to be here.

Reply:
Is it possible to copy this (and all the other big ../../../) into a normalized /opt or /etc dir for sourcing to avoid that big relative path? We add one directory to this chain in the future and every script is broken.

Reply:
As discussed on Slack, these are development-only scripts that just facilitate the process of building images against a local Docker but then pushes the images to a Docker registry running in Openshift. They're not in use in any production image.